acceptance_tests/features/steps/action_steps.py

```
@@ -14,6 +14,8 @@ def setup_action_rule(context, action_type, action_rule_delay):
 
     trigger_date_time = (datetime.utcnow() + timedelta(seconds=int(action_rule_delay))).isoformat() + 'Z'
 
+    classifiers = None
+
```

This set of ifs and functions just serves to map strings to dictionaries so could be done simpler like this:
```python
classifiers_for_action_type = {
        'ICL1E': {'treatmentCode': ['HH_LF3R2E', 'HH_LF3R3AE', 'HH_LF3R3BE', 'HH_LFNR1E', 'HH_LF2R3BE']},
        'ICL2W': {'treatmentCode': ['HH_LFNR1W', 'HH_LFNR2W', 'HH_LFNR3AW', 'HH_LF2R1W', 'HH_LF2R2W', 'HH_LF2R3AW',
                                    'HH_LF2R3BW', 'HH_LF3R1W', 'HH_LF3R2W', 'HH_LF3R3AW', 'HH_LF3R3BW']},
        'ICHHQW': {'treatmentCode': ['HH_QF2R1W', 'HH_QF2R2W', 'HH_QF2R3AW', 'HH_QF3R1W', 'HH_QF3R2W', 'HH_QF3R3AW']}
    }

    create_action_rule(str(uuid.uuid4()), trigger_date_time,
                       classifiers_for_action_type.get(action_type), action_plan_url, action_type)
```

---

I agree with @AdamHawtin though we might want to fail hard if the key isn't found:

```python
classifiers_for_action_type[action_type]
```

instead of the `.get`

---

@jcox-dev Agreed, that would prevent unexpectedly running it with no classifiers by erroring out instead. That would just require added a `None` entry to the dictionary for any action type we don't want classifiers for, .e.g
```python
classifier_for_action_type = {
        'ICL1E': {'treatmentCode': ['HH_LF3R2E', 'HH_LF3R3AE', 'HH_LF3R3BE', 'HH_LFNR1E', 'HH_LF2R3BE']},
        'ICL2W': {'treatmentCode': ['HH_LFNR1W', 'HH_LFNR2W', 'HH_LFNR3AW', 'HH_LF2R1W', 'HH_LF2R2W', 'HH_LF2R3AW',
                                    'HH_LF2R3BW', 'HH_LF3R1W', 'HH_LF3R2W', 'HH_LF3R3AW', 'HH_LF3R3BW']},
        'ICHHQW': {'treatmentCode': ['HH_QF2R1W', 'HH_QF2R2W', 'HH_QF2R3AW', 'HH_QF3R1W', 'HH_QF3R2W', 'HH_QF3R3AW']},
        'FF2QE': None
    }
```